apps/web/src/app/host-dashboard/page.tsx

```
@@ -0,0 +1,1118 @@
+                    <label htmlFor="location" className="block text-sm dark:text-white font-medium text-gray-700 mb-2">Location</label>
+                    <input 
+                    id='location'
+                      type="text" 
+                      placeholder="City, State"
+                      className="w-full px-3 py-2 dark:text-white border border-gray-300 rounded-lg focus:ring-0 bg-transparent focus:ring-blue-500 focus:border-transparent"
+                    />
+                  </div>
+                </div>
+
+                <div className="mt-6">
+                  <label className="block dark:text-white text-sm font-medium text-gray-700 mb-2">Bio</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)